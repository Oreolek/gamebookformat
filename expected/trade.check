examples/trade.json: Could not reach section 'someotherplace' from start.
