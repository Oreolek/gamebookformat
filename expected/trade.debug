BEGIN DEBUG OUTPUT
Book title: Gamebook
Number of sections: 2
Introduction
 Testing trade of things. You can buy things using [COUNT gold]Gold[/COUNT] of which you start with [SET gold]100[/SET]. You also have an [COLLECT item]Inventory[/COLLECT]. At start you carry a [ADD item]sword[/ADD]. 


Turn to 1 to begin.
1 (start) -  You can buy a [FOUND item]shield[/FOUND] here for 50 Gold. There is also a [FOUND item]chain mail[/FOUND] costing 150. You can pick up some [FOUND item]spiderweb[/FOUND] for free. You can go on to the end at 2. 
2 (theend) -  It ends here.
END DEBUG OUTPUT
