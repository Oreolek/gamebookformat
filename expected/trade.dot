digraph gamebook {

 shield50chain mail150spiderweb1->2
}
