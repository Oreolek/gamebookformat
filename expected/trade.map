{"start": 1, "other": 4, "theend": 2, "someotherplace": 3}