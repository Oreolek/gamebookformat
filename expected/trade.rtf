{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Gamebook
\b0\
\
\b \qc Introduction
\b0\



\ql  Testing trade of things. You can buy things using Gold of which you start with 100. You also have an Inventory. At start you carry a sword.  \
\
\b Turn to 1 to begin.
\b0\
\
\b \qc 1
\b0\
\ql  You can buy a shield here for 50 Gold or a beautiful gold ring for only 75. There is also a chain mail costing 150. You can pick up some spiderweb for free. You can go to \b 4
\b0
 at a cost of 50. You can go on to the end at \b 2
\b0
.  \
\

\b \qc 2
\b0\
\ql  It ends here. There is a crown here that you can trade for some spiderweb.  \
\

\b \qc 3
\b0\
\ql  This is somewhere else. Congratulations you have found 100 gold! Go back to \b 1
\b0
.  \
\

\b \qc 4
\b0\
\ql  Now you have 50 less gold than you just had. You can go back to \b 1
\b0
.  \
\

}
