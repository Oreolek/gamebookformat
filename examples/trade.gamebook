= Introduction
Testing trade of things. You can buy things using
[count gold]Gold[/count] of which you start with [set gold]100[/set].
You also have an [collect item]Inventory[/item]. At start you carry
a [add item]sword[/add].

* 1 start
You can buy a [found item]shield[/found] here for [cost gold]50[/cost] Gold
or a beautiful [found item]gold ring[/found] for only [cost gold]75[/cost].
There is also a [found item]chain mail[/found] costing [cost gold]150[/cost].
You can pick up some [found item]spiderweb[/found] for free.
You can go to [[other]] at a cost of [cost gold]50[/cost].
You can go on to the end at [[theend]].

* other
Hopefully you now have 50 less gold than you just had.
You can go back to [[start]].

* theend
It ends here.
