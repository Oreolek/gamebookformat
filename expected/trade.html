<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Gamebook</title>
    <script type="text/javascript" src="gamebookformatplay.js"></script>
    <link rel="stylesheet" href="gamebookformat.css"
          type="text/css" />
    <script>
      if (typeof gamebook !== 'undefined') {
         gamebook.id = 'trade';
      }
    </script>
  </head>
  <body>
<div class="hideintrolink nodisplay"
    onclick="gamebook.hideIntroSections()">(hide instructions)</div>
  <div class="gamebook">
<div class="introsection">
<div class="introsectionheading">Introduction</div>
<div class="introsectionbody">
 Testing trade of things. You can buy things using <span class="count counterChange" data-type="gold"
    data-name="Gold">Gold</span> of which you start with <span class="set counterChange" data-type="gold"
      data-amount="100">100</span>. You also have an <span class="collect" data-type="item"
    data-name="Inventory">Inventory</span>. At start you carry a <span class="add" data-type="item"
      data-what="sword">sword</span>. 
</div>
</div>

<div class="resumelink nodisplay"
  onclick="gamebook.loadGame()">Resume saved game.</div>

<div class="startlink"
  onclick="gamebook.turnTo(1)">Turn to 1 to begin.</div>
<script>
    if (typeof gamebook !== 'undefined' && gamebook.hasSavedGame()) {
        var resumeLinks = document.getElementsByClassName('resumelink');
        Array.prototype.forEach.call(resumeLinks, function(e) {
            e.classList.remove('nodisplay');
        });
    }
</script>
<div class="section" id="section1">
<div class="sectionnumber" id="para1">1</div>
<div class="sectiontext">
 You can buy a <span class="found enabledlink" data-type="item"
      data-what="shield"
>shield</span>
 here for <span class="cost enabledlink" data-type="gold"
      data-amount="50"
>50</span>
 Gold. There is also a <span class="found enabledlink" data-type="item"
      data-what="chain mail"
>chain mail</span>
 costing <span class="cost enabledlink" data-type="gold"
      data-amount="150"
>150</span>
. You can pick up some <span class="found enabledlink" data-type="item"
      data-what="spiderweb"
>spiderweb</span>
 for free. You can go on to the end at <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(1, document.getElementById('section1'));
}
</script><div class="section" id="section2">
<div class="sectionnumber" id="para2">2</div>
<div class="sectiontext">
 It ends here.
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(2, document.getElementById('section2'));
}
</script>     <div id="counters" class="counters">
    </div>
    <div id="counterTemplate" class="counterTemplate">
      <span class="counterheading"></span>
      <span class="countercontents"></span>
    </div>
    </div>
    <div id="collections" class="collections">
    </div>
    <div id="collectionTemplate" class="collectionTemplate">
      <span class="collectionheading"></span>
      <span class="collectioncontents"></span>
    </div>
    </div>
    <script>
if (typeof gamebook !== 'undefined') {
   gamebook.prepare();
}
    </script>
<div class="displayintrolink nodisplay"
    onclick="gamebook.showIntroSections()">(show instructions)</div>
    </div>
  </body>
</html>
